### ISSUE TITLE
Error in `PUT`ting `/api/v1/regular_invest`

### ISSUE THREAD

**Issue by wyhfrank** [Chen33D17017/auto-invest-spring#15]
This should not return an invalid token error.

``` json
{
    "data": "Request processing failed; nested exception is org.springframework.dao.CannotAcquireLockException: could not execute statement; SQL [n/a]; nested exception is org.hibernate.exception.LockAcquisitionException: could not execute statement",
    "resultInfo": {
        "codeId": "010000009",
        "code": "INVALID_TOKEN",
        "message": "invalid token",
        "status": "F"
    }
}
```

**Comment by wyhfrank** [comment]
Similar error happens when `POST`ing to `/api/v1/regular_invest` where the record already exists.

``` json
{
    "data": "Request processing failed; nested exception is org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL [n/a]; constraint [UniqueUserIdAndCryptoName]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement",
    "resultInfo": {
        "codeId": "010000009",
        "code": "INVALID_TOKEN",
        "message": "invalid token",
        "status": "F"
    }
}
```

### Expected behavior

`PUT` and `POST` simply overwrites the records with the same `cryptoName`.

**Comment by Chen33D17017** [comment]
@wyhfrank Could you tell me how to reproduce the first one?

**Comment by wyhfrank** [comment]
> @wyhfrank Could you tell me how to reproduce the first one?

Only occurs when using web app. Multiple calls are made (almost) at the same time, which might be the cause.

**Comment by Chen33D17017** [comment]
It is because the read lock for the db, the db will be failing to update the db when the specific row had been locked

**Comment by Chen33D17017** [comment]
Will fix with #20
